.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
             matrix:
                 php-version: [
                     '7.4',
-                    '8.1'
+                    '8.2'
```

Comment:
in this case we don't check 8.1. but I think it's should ok

Reply:
The general concept is to test minimum supported version + the latest one. That's why we won't test 8.0 and 8.1

Reply:
But the problem is that Laminas packages don't have such versions that work with php 7.4 and with 8.2 at the same time. So I keep 8.1 here and as Pavlo did - update Laminas packages for 8.2 image